src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -9585,6 +9918,96 @@ public override object PayloadValue(int index)
         #endregion
     }
 
+    public sealed class GCSettingsRundownTraceData : TraceEvent
+    {
+        public long HardLimit { get { return GetInt64At(0); } }
+        public long LOHThreshold { get { return GetInt64At(8); } }
+        public long PhysicalMemoryConfig { get { return GetInt64At(16); } }
+        public long Gen0MinBudgetConfig { get { return GetInt64At(24); } }
+        public long Gen0MaxBudgetConfig { get { return GetInt64At(32); } }
+        public int HighMemPercentConfig { get { return GetInt32At(40); } }
+        public int BitSettings { get { return GetInt32At(44); } }
```

Comment:
we should include the definition of bits in this BitSettings field since this is a contract (we don't change the meaning of the bits), like MarkRootType